alburnum/maas/viscera/users.py

```diff
@@ -20,6 +20,12 @@ class UsersType(ObjectType):
     def __iter__(cls):
         return map(cls._object, cls._handler.read())
 
+    def create(cls, username, email, password, *, is_superuser=False):
```

Comment:
`email` is a vestigial field that we don't actually use, but we have to supply. I think this method should either:

- Auto-generate an email address like `${username}@null.maas.io`.

- Put `email` back to the third parameter after `password`.

- A bit of both: make `email` an optional keyword-only parameter, and auto-generate an address if it's not given.